### ISSUE TITLE
Telega error 406: UPDATE_APP_TO_LOGIN

### ISSUE THREAD

**Issue by nanounanue** [zevlg/telega.el#329]
Telega error 406: UPDATE_APP_TO_LOGIN

## Telega Setup
*OS*: Pop!_OS 21.04
*Emacs*: GNU Emacs 28.0.60 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.24.24, cairo version 1.16.0)
*Telega*: telega v0.7.030 (TDLib v1.7.0) (telega-server v0.7.5)

## Current Behavior

When I execute 'M-x telega` I got the following error: `Telega error 406: UPDATE_APP_TO_LOGIN`
## Steps to Reproduce

1. M-x telega
2. Introduce my phone number
3. Got the mentioned error

## Telega Setup
*OS*: 
*Emacs*: 
*Telega*: 

## Steps to Reproduce

1.
2.
3.

**Comment by zevlg** [comment]
Uptade your telega to latest from master branch

**Comment by nanounanue** [comment]
I am using the version from `guix`, do you know if the most recent version in the guix repo has that already?

**Comment by zevlg** [comment]
🤷🏻 I doubt, but installation directly from github will work

**Comment by nanounanue** [comment]
Thanks! I will do that then

**Comment by WorldsEndless** [comment]
I can confirm: after installing manually using the instructions on their repo, it worked. I tried twice with the Telega documentation instructions and kept on getting my old, failing version. The difference, assuming you are operating in the cloned repo that is the same either way, is that they use a cmake directive instead of the make on in the telega documentation. I also went without the `-J#` part to make use of my cores and so it took forever, but here is the working build command which, in conjunction with using the latest Releases branch code, got me loading without Telegram itself complaining. 

```
cmake -DCMAKE_BUILD_TYPE=Release ..
cmake --build .
```

it was the `DCMAKE_BUILD_TYPE` bit that seemed to make the difference (and note that these official things use cmake instead of the plain make at https://zevlg.github.io/telega.el/index-master.html#building-tdlib . 

Now I am back to #316 which I never really solved, but that's a different story. Point is, Telegram itself seems okay with me now.
